Makefile

```diff
@@ -0,0 +1,27 @@
+.PHONY: conda-update
+conda-update:
+	conda env update -n saturn-client --file environment.yaml
+
+.PHONY: format
+format:
+	black --line-length 100 .
+
+.PHONY: lint
+lint:
+	flake8 --count --max-line-length 100 --exclude versioneer.py .
+	black --check --diff --line-length 100 .
+	# pylint disables:
+	#   * C0301: line too long
+	#   * C0103: snake-case naming
+	#   * C0330: wrong hanging indent before block
+	#   * E0401: unable to import
+	#   * R0903: too few public methods
+	#   * R0913: too many arguments
+	#   * R0914: too many local variables
+	#   * R1705: Unnecessary "else" after "return"
+	#   * R1720: Unnecessary "elif" after "return"
+	#   * W0107: unnecessary "pass" statement
+	#   * W0212: access to protected member
+	#   * W0221: parameters differ from overridden method
+	#   * W1203: use lazy % formatting
+	pylint --disable=C0103,C0301,C0330,E0401,R0903,R0913,R0914,R1705,R1720,W0107,W0212,W0221,W1203 saturn_client/
```

Comment:
was this copied from `dask-saturn`? If so it's something I wrote before I really understood `pylint`, and I don't think it's a good pattern.

I think we should have a preference for using a `.pylintrc` fie instead, since it will then control `pylint` no matter how you run it (`make lint`, `pylint saturn_client/`, `pre-commit`, etc.)

Reply:
yep. Switching to that.